controllers/provisioners/eks/helpers.go

```diff
@@ -302,6 +302,10 @@ func (ctx *EksInstanceGroupContext) GetAddedTags(asgName string) []*autoscaling.
 		switch strings.ToLower(osFamily) {
 		case OsFamilyWindows:
 			tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/label/kubernetes.io/os", "windows", asgName))
+			tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/label/kubernetes.io/arch", "amd64", asgName))
+			instanceTypeNetworkInfo := awsprovider.GetInstanceTypeNetworkInfo(ctx.GetDiscoveredState().GetInstanceTypeInfo(), configuration.InstanceType)
```

Comment:
We also call this in `GetComputedBootstrapOptions`, should we simply make sure this call is made in CloudDiscovery, and then stored in DiscoveredState?

Reply:
yep - good call

Reply:
Oh NVM, I thought this was an API call, looks like it's just a helper function 👍 
In this case can we just move `ctx.GetDiscoveredState().GetInstanceTypeInfo()` to top var declaration for order/OCD purposes 😀